edwintantawi/pseudocode.txt

```diff
@@ -0,0 +1,12 @@
+BEGIN
+	LIST destinations
+	INPUT userDestination
+	IF isDestinationAvailable(userDestination)
+		INPUT userSchedule
+		IF isScheduleAvailable(userSchedule)
+			booking()
+		ELSE
+			OUTPUT error("schedule is not available")
+	ELSE
+		OUTPUT error("destination is not available")
```

Comment:
change error to "please change destination, destination you choice is not available"